Packs/CommonScripts/Scripts/HttpV2/README.md

```diff
@@ -0,0 +1,134 @@
+Sends http request
```

Comment:
Let's expand this to include more data about the capabilities here, from a bird's eye view